supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -1079,6 +1079,21 @@ Use the name of the tab when you refer to it, for example, "the *Storage* tab".
 
 *See also*:
 
+[[rulebook]]
+==== image:images/yes.png[yes] rulebook (noun)
+*Description*: A _rulebook_ is a YAML file containing a list of rules, event source definitions, and hosts to run against. 
+Rulebooks are used in Event-Driven Ansible. When using the term _rulebook_ without the Ansible prefix, use lowercase _r_.
+
+Examples:
+
+* Use a rulebook in Ansible.
```

Comment:
The "playbook" entry has the same two examples as the "Ansible Playbook" entry.

However, the "rulebook" entry has only one example. Should we add the other example from the "Ansible Rulebook" entry?

Reply:
I would say it's fine to leave the one example because the `Ansible Rulebook` usage is not mentioned in the Description. In the other entry, both usages are mentioned in the description. I suppose you could expand the description here so that it's parallel with the other entry, and then add the extra example. But if you leave it as-is, then imo no extra example is needed.

Reply:
It wouldn't hurt to add it, per @bredamc's suggestion, because the two usages are closely related.